### DISCUSSION TITLE
Getting an issue while installing Schema Registry

### DISCUSSION THREAD

**Discussion by Muddam234** [strimzi/strimzi-kafka-operator#7830] [Q&A]
Hi All,

Can someone please help me on the below mention issue.

Error: INSTALLATION FAILED: unable to build kubernetes objects from release manifest: resource mapping not found for name: "qa-schema-registry" namespace: "" from "": no matches for kind "Ingress" in version "extensions/v1beta1"
ensure CRDs are installed first

**Comment by scholzj** [comment]
Strimzi does not support any Schema Registry. So the context is not really clear. The error suggests you are installing some application designed for older Kubernetes versions on a Kubernetes lcuster running newer Kubernetes version which oes not support anymore Ingress `extensions/v1beta1`. You would need to update the Ingress resource to use a newer API.
